cdap-examples/WikipediaPipeline/pom.xml

```diff
@@ -35,8 +35,7 @@
   <dependencies>
     <dependency>
       <groupId>co.cask.cdap</groupId>
-      <artifactId>cdap-api</artifactId>
-      <version>${project.version}</version>
+      <artifactId>cdap-api-spark2_2.11</artifactId>
```

Comment:
Prior to your PR, the WikipediaPipeline app was still compatible with Spark2. With these changes, is it no longer compatible with Spark1?

Reply:
It's not the intention to keep it compatible with Spark 1, @sreevatsanraman right?

Reply:
Let's call it out in the docs if they are not compatible. Since the component is 2+ years old, we don't need to maintain this compat for all examples.